go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/maruel/natural
 
 go 1.11
+
+require github.com/facette/natsort v0.0.0-20181210072756-2cd4dd1e2dcb
```

Comment:
Adding a dependency on another natural sorting library doesn't make sense to me. Even if it is only used by tests.
This would lead to a circular dependency if both libraries would want to test each other.

Is there a reason why this comparative benchmark cannot live in a separate repository?

Reply:
It's a really good point, I didn't think of this kind of issue. Would it make sense to you in the current state?

Benchmarks aren't launchable from the repository root, but no conflicts